packaging/distributions-dependencies/build.gradle.kts

```diff
@@ -98,7 +98,8 @@ dependencies {
         api(libs.groovyXml)             { version { strictly(libs.groovyVersion) }}
         api(libs.gson)                  { version { strictly("2.10") }}
         api(libs.h2Database)            { version { strictly("2.2.220") }}
-        api(libs.hamcrestCore)          { version { strictly("1.3"); because("2.x changes the API") }}
+        api(libs.hamcrest)              { version { strictly("3.0") }}
+        api(libs.hamcrestCore)          { version { strictly("3.0") }}
```

Comment:
💅 I suppose `hamcrestCore` can be removed completely? There are only two usages

Reply:
https://github.com/gradle/gradle/pull/32740#discussion_r2003743373